public/app/core/components/PageNew/SectionNavItem.tsx

```diff
@@ -24,7 +24,7 @@ export function SectionNavItem({ item, isSectionRoot = false }: Props) {
   const linkClass = cx({
     [styles.link]: true,
     [styles.activeStyle]: item.active,
-    [styles.isSection]: Boolean(children?.length),
+    [styles.isSection]: Boolean(children?.length) || item.isSection,
```

Comment:
Would it make sense to add a test for this scenario? Maybe not so much for verifying but more for the sake of documentation and explaining why this is needed?